src/utils/pii-remover.ts

```diff
@@ -1,6 +1,6 @@
 const EMAIL_PATTERN = /[^\s=/?&#+]+(?:@|%40)[^\s=/?&+]+/g;
 const POSTCODE_PATTERN =
-  /\b[A-PR-UWYZ][A-HJ-Z]?[0-9][0-9A-HJKMNPR-Y]?(?:[\s+]|%20)*[0-9](?!refund)[ABD-HJLNPQ-Z]{2,3}\b/gi;
+  /\b[A-PR-UWYZ][A-HJ-Z]?\d[0-9A-HJKMNPR-Y]?(?:[\s+]|%20)*\d(?!refund)[ABD-HJLNPQ-Z]{2,3}\b/gi;
```

Comment:
are you sure about this change? 
Why do we have to update the REGEXP? and what is the difference between the existing one and your new one?

Reply:
It should make no functional difference, apparently its the same thing, just a more concise way of doing it.

Link to the [issue](https://sonarcloud.io/project/issues?open=AYx9OYF8zDN-BIKbnAfi&id=govuk-one-login_di-fec-ga4)